swagger.yaml

```diff
@@ -116,7 +118,60 @@ paths:
           description: Wrong content type (should be JSON)
         422:
           description: Missing fields
-
+  /projects:
+    get:
+       tags:
+        - projects 
+      summary: Get project list
+      description: Returns a list of project for a specific user 
+      security:
+        - user_id: []
+      responses:
+        200:
+          description: Projects list 
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Projects"
+        404:
+          description: Project not found
+        500:
+          description: Internal server error
```

Comment:
I didn't document 500s earlier, since they're not technically part of the API specification.

Reply:
Okey.